frontend/src/constants/datasets.ts

```diff
@@ -3692,6 +3692,6 @@ export let DATASETS: Dataset[] = [
             lng: -90.175626
         }
     ],
-    serviceAreaIds: MississippiServiceAreas
+    serviceAreaIds: SERVICE_AREAS_BY_STATE['ms']
```

Comment:
In JS, it's good style to use dot notation over bracket notation when possible. Eg.

```diff
- serviceAreaIds: SERVICE_AREAS_BY_STATE['ms']
+ serviceAreaIds: SERVICE_AREAS_BY_STATE.ms
```

If the linter isn't catching this already, we should add a rule for it.